cmd/minishift/cmd/util/util.go

```diff
@@ -149,3 +152,12 @@ func ValidateGenericDriverFlags(remoteIPAddress, remoteSSHUser, sshKeyToConnectR
 		atexit.ExitWithMessage(1, fmt.Sprintf("Error: %s", msg))
 	}
 }
+
+// OcClusterDown down the cluster and if there is any issue during the minishift start before even
+// oc binary cached then it just ignore the error [PK]
+func OcClusterDown(hostVm *host.Host) error {
+	sshCommander := provision.GenericSSHCommander{Driver: hostVm.Driver}
+	cmd := fmt.Sprintf("%s/oc cluster down", minishiftConstants.OcPathInsideVM)
+	sshCommander.SSHCommand(cmd)
```

Comment:
Why are we not catching the error here? As per function definition  we should return error but we are not doing it.